### PR TITLE
Adds green jester shoes for the green jester outfit.

### PR DESCRIPTION
**Pull request by Slorbin**
## About The Pull Request

I went back and did a few fixes for GBP to do this.
This adds a set of green jester shoes to the game and to the autodrobe, since for some reason, while we HAVE jester shoes in the game, it's only for the alternative set of yellow jester clothes. These shoes don't waddle but they do jingle a bit.
## Why It's Good For The Game

There are 0 good shoes that go with the green jester outfit since it's lacking an entire third of the costume for some inconceivable reason. Here are some shoes that go with it and jingle.
## Changelog
:cl:
add: Adds green jester shoes that jingle to the autodrobe!
/:cl:

### CODE REVIEW

**Comment by Wallemations** [comment]
image


?

**Review by Wallemations — CHANGES_REQUESTED** [review]
Fix those icon states

**Comment by DeerJesus** [comment]
That’s odd, it’s not doing that at all on my end. I’ll try that again.

**Comment by DeerJesus** [comment]
> Fix those icon states

fixed. apparently i hit save in dream maker editor and save in vscode and it just overrid me fixing the shoes earlier my bad. fix'd

**Review by Wallemations — APPROVED** [review]
Am I allowed to approve sounds? I don't know. The sprites look fine though

**Review by Ghommie — CHANGES_REQUESTED** [review]
The sound file isn't mono and the project rate is 48000, not 44100. The file is already an .ogg however.
You can easily fix that in Audacity. [Here are the instuctions](https://github.com/tgstation/tgstation/blob/master/sound/standard.md).

**Comment by DeerJesus** [comment]
Thanks ghommie! I’ll have that fixed in a jiffy

**Comment by Ghommie** [comment]
Oh, by the by, I'd assume the sound is public domain (cc 0), but if not then an attribution should be included in the attributions text file inside the sounds folder.

**Comment by DeerJesus** [comment]
> Oh, by the by, I'd assume the sound is public domain (cc 0). If not then an attribution should be included in the attributions text file inside the sounds folder.

I added an attribution.

**Comment by DeerJesus** [comment]
> The sound file isn't mono and the project rate is 48000, not 44100. The file is already an .ogg however. You can easily fix that in Audacity. [Here are the instuctions](https://github.com/tgstation/tgstation/blob/master/sound/standard.md).

This is now done.

File: sound/attributions.txt [hunk 1]
```diff
@@ -87,3 +87,7 @@ https://freesound.org/people/squareal/sounds/237375/
 
 ding_short.ogg is from Natty23 (CC 4)
 https://freesound.org/people/Natty23/sounds/411747/
+
+
+jingle.ogg is from Zapsplat (CC 4)
```
Comment (Ghommie) [inline sound/attributions.txt:92]:
```suggestion
jingle.ogg is from Zapsplat (https://www.zapsplat.com/license-type/standard-license/)
```
It doesn't say it's actually CC 4, though it's similar.

**Pull request merged**